core/src/main/java/io/questdb/cutlass/http/client/HttpClient.java

```diff
@@ -438,6 +438,14 @@ public Request setCookie(CharSequence name, CharSequence value) {
             return this;
         }
 
+        public int getContentLen() {
```

Comment:
Confusingly there seems to be another method called `getContentLength()` used around here in this file.
Are they different? This might be a possible rename candidate. How about `getRequestByteSize()`?

Reply:
where do you see the existing method? 😕

Reply:
the existing `getContentLength()` is on received response headers. It gives you context length of the received response. 

While this new method in on a request. It gives you the content length of the request. 
 
They should both stay, but I will align their names.